frontend/src/main/pages/LeaderboardPage.js

```diff
@@ -10,12 +10,20 @@ import { useBackend } from "main/utils/useBackend";
 import { useCurrentUser } from "main/utils/currentUser";
 import Background from "../../assets/PlayPageBackground.png";
 
+//#12
```

Comment:
```suggestion
```

It no longer considered good practice to clutter the code with issue or PR numbers in comments like this, since GitHub tracks all of that for us.   Can we remove this?